napture/src/main.rs

```diff
@@ -374,10 +411,14 @@ fn fetch_dns(url: String) -> String {
         let status = response.status();
 
         if let Ok(json) = response.json::<DomainInfo>() {
-            json.ip
+            return json.ip
```

Comment:
Why did you add return here ?

Reply:
Did that out of habit of other languages lol